scripts/vm/network/security_group.py

```diff
@@ -1029,6 +1044,14 @@ def addFWFramework(brname):
         sys.exit(1)
     cmd = args[0]
     logging.debug("Executing command: " + str(cmd))
+
+    for i in range(0,30):
```

Comment:
we can replace doing 30 loops and the if <condition>, with a -- with (resource): do stuff ...